### ISSUE TITLE
Add support for @sentry/fullstory

### ISSUE THREAD

**Issue by tjhiggins** [nuxt-community/sentry-module#416]
### Describe the solution you'd like
Add support for fullstory integration. Either as a supported integration or via the custom integrations pr https://github.com/nuxt-community/sentry-module/pull/276.

I'd be willing to put up a pr to add support for fullstory. It could be one of those integrations where its required you install the package out of band to use. 

```js
clientIntegrations: {
  Fullstory: {} // Requires `npm install @sentry/fullstory`
},
```

### Additional context
https://github.com/getsentry/sentry-fullstory

**Comment by rchl** [comment]
I don't remember exactly what is holding up #276 but that would be the way to go IMO.

**Comment by tjhiggins** [comment]
> I don't remember exactly what is holding up #276 but that would be the way to go IMO.

That works for me! Think you'd have the time to take another look? Pairing sentry/fullstory seems pretty amazing and would love to try it out.

**Comment by tjhiggins** [comment]
@rchl what could I do to help? Would prefer to keep using this package. I can have one of my engineers pull down that pr and make sure it still works with the latest.

**Comment by rchl** [comment]
Took a bit longer than a week but I've now released it in beta version - `npm i @nuxtjs/sentry@beta`.

**Comment by mammuth** [comment]
FYI I'm not sure whether just adding the Fullstory Integration to the nuxt config works as expected, as the Sentry<>Fullstory integration only works after fullstory was initialized (and the recording started).

It for sure didn't work for me, when also initializing Fullstory "later", eg. after consent was given for the privacy policy.

I solved it like this (keeping it here for the future reader):

```javascript
FullStory.init({ orgId: FULLSTORY_ORG_ID }, fullStoryCallback);

const fullStoryCallback = ({ sessionUrl }) => {
  const currentSentryClient = this.$sentry.getCurrentHub().getClient();
  const sentryOptions = currentSentryClient.getOptions();
  sentryOptions.integrations = [...sentryOptions.integrations, new SentryFullStory(SENTRY_ORG_SLUG)];
  const newSentryClient = new Sentry.BrowserClient(sentryOptions);
  this.$sentry.getCurrentHub().bindClient(newSentryClient);
}
```

The idea to rebind the sentry client was taken from https://github.com/getsentry/sentry-javascript/issues/2160

**Comment by rchl** [comment]
Well, according to https://github.com/getsentry/sentry-fullstory#setup instructions the integration can be set up before FullStory has initialized. So if that doesn't work then maybe it should be raised in their repo.

Have you actually tried it with the beta build?

**Comment by mammuth** [comment]
> Have you actually tried it with the beta build?

Yes, I tried it right after you merged your PR.

However, I had a wrong assumption for the reason why it didn't work :facepalm: 
I continued seeing `current session URL API not ready` in the events in Sentry. But this was due to the configuration in Fullstory which didn't whitelist the hostname I was testing it on, I think.

**TL;DR: Just adding the custom integration works like a charm.** :100:
